config/config.yaml

```
@@ -19,6 +19,10 @@ redis:
   port: 6379
   token: my-sup3r-comp1ic4t3d-s3cr3t-t0k3n
 
+defaults:
+  - subscription_id: notreal-default-subscription
+    resource_group: notreal-default-rg
```

```suggestion
defaults:
  subscription_id: notreal-default-subscription
  resource_group: notreal-default-rg
```
Don't think this needs to be an array/list/slince whatever it is :D